applications/NXxas_new.nxdl.xml

```diff
@@ -0,0 +1,230 @@
+                        TODO
+                    </doc>
+                </item>
+                <item value="electron energy loss">
+                    <doc>
+                        TODO
+                    </doc>
+                </item>
+                <item value="dafs">
+                    <doc>
+                        TODO
+                    </doc>
+                </item>
+                <item value="other">
+                    <doc>
+                        TODO
+                    </doc>
+                </item>
+            </enumeration>
+        </field>
+        <group type="NXelement" name="element">
+            <doc>Excited element</doc>
+            <field name="symbol"></field>
+        </group>
+        <group type="NXelectron_level" name="absorption_edge">
+            <doc>Excited electron level</doc>
+            <field name="name"></field>
+        </group>
+        <field name="emission_line" type="NX_CHAR" optional="true">
+            <doc>
+                Emission lines relevant when :ref:`mode &lt;/NXxas_new/ENTRY/mode-field&gt;` is
+                `fluorescence yield` or `herfd`.
+
+                The emission lines are described as a list of names of the form
+                *"INITIAL-FINAL"* where *INITAL* and *FINAL* are electron levels
+                as listed for :ref:`here &lt;/NXelectron_level/name-field&gt;`.
+            </doc>
+            <dimensions rank="1">
+                <dim index="1" value="nTransitions"/>
+            </dimensions>
+        </field>
+        <field name="energy" type="NX_FLOAT" units="NX_ENERGY">
+            <doc>TODO</doc>
+            <dimensions rank="1">
+              <dim index="1" value="nEnergy" />
+            </dimensions>
+        </field>
+        <field name="intensity" type="NX_FLOAT" units="NX_ANY">
+            <doc>TODO</doc>
+            <dimensions rank="1">
+              <dim index="1" value="nEnergy" />
+            </dimensions>
+        </field>
+        <field name="intensity_errors" type="NX_FLOAT" units="NX_ANY">
+            <doc>TODO</doc>
+            <dimensions rank="1">
+              <dim index="1" value="nEnergy" />
+            </dimensions>
+        </field>
+        <group type="NXsample">
+            <field name="name">
+                <doc>Descriptive name of the sample</doc>
+            </field>
+        </group>
+        <group type="NXprocess" optional="true">
+            <doc>
+                Description on how :ref:`energy &lt;/NXxas_new/ENTRY/energy-field&gt;`
+                and :ref:`intensity &lt;/NXxas_new/ENTRY/intensity-field&gt;` were obtained
+                from the raw data.
+            </doc>
+        </group>
+        <group type="NXinstrument" optional="true">
+            <group type="NXsource">
+                <field name="type"/>
+                <field name="name"/>
+                <field name="probe">
+                    <enumeration>
+                        <item value="x-ray"/>
```

Comment:
If you want to allow the "electron energy loss" mode, then you should also allow an "electron" probe type.